querynator/query_api/civic_api.py

```diff
@@ -498,23 +510,35 @@ def concat_dicts(coord_id_dict, variant_obj, filter_vep):
         return {**coordinates_info, **variant_info, **gene_info, **mol_profile_info, **assertion_info, **evidence_info}
 
 
-def create_civic_results(variant_list, out_path, logger, filter_vep):
+def create_civic_results(variant_list, out_path, disease, logger, filter_vep):
     """
     Combine result dictionaries of all CIViC variant objects
     to a table and write it to user-specified file
 
     :param variant_list: List of CIViC variant objects of successfully queried variants
-    :type variant_list: list
+    :type  variant_list: list
     :param out_path: Name for directory in which result-table will be stored
-    :type out_path: str
+    :type  out_path: str
+    :param disease: the patients cancer type as Disease Ontology Name
+    :type  disease: str
     :param filter_vep: flag whether VEP based filtering should be performed
-    :type filter_vep: bool
+    :type  filter_vep: bool
     :return: None
     :rtype: None
     """
     civic_result_df = pd.DataFrame()
+
+    if disease:
+        doid = ontology.DO("querynator/helper_functions/doid.obo")
+        diseases = doid.get(disease), doid.get_all_ancestors(disease)
+        logger.info(f"Mapped specified disease {disease} to Disease Ontology {str(diseases[0])}")
```

Comment:
```suggestion
        logger.info(f"Mapped specified disease {disease} to Disease Ontology (DO) {str(diseases[0])}")
```
Just to make it a bit more explicit because the class is called DO

Reply:
I thought about it. Might be easier to debug somewhen later if it has a more descriptive name. So I would change it.